### PR TITLE
feat: date time input

### PR DESCRIPTION
**Pull request by AasmundN**
# Changes 

- Add date time input component.
- Update event and job edit pages accordingly.

### CODE REVIEW

File: composables/useDate.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
 export const isValidDate = (date: string): boolean => {
   // Check if date matches date regex
   const dateRegex = /^\d{4}-\d{2}-\d{2}$/
-  if (!dateRegex.test(date)) {
+  if (!dateRegex.test(date.split('T')[0])) {
```
Comment (AasmundN) [inline composables/useDate.ts:5]: Hope this doesn't break anything 🤞

Comment (JStoresund) [reply composables/useDate.ts:5]: Makes sense to me

**Review by JStoresund — CHANGES_REQUESTED** [review]
Styling looks good

For some reason, there is no clock UI on firefox. It works nicely on the chrome-app on my phone.

I suggest looking into it a bit, since companies might be using firefox to create/update event or job information, and they will most certainly be annoyed by the issue.

Apart from this, it looks good

**Review by JStoresund — APPROVED** [review]
LGTM

**Pull request merged**